docs/src/pages/components/buttons/LoadingButtonsTransition.tsx

```diff
@@ -1,32 +1,30 @@
 import * as React from 'react';
-import { makeStyles } from '@material-ui/core/styles';
+import { experimentalStyled as styled } from '@material-ui/core/styles';
 import LoadingButton from '@material-ui/lab/LoadingButton';
-import FormControlLabel from '@material-ui/core/FormControlLabel';
+import Box from '@material-ui/core/Box';
+import MuiFormControlLabel from '@material-ui/core/FormControlLabel';
 import Switch from '@material-ui/core/Switch';
 import SaveIcon from '@material-ui/icons/Save';
 import SendIcon from '@material-ui/icons/Send';
 
-const useStyles = makeStyles((theme) => ({
-  root: {
-    '& button': {
-      margin: theme.spacing(1),
-    },
-  },
-  switch: {
-    display: 'block',
-  },
-}));
+const FormControlLabel = styled(MuiFormControlLabel)(`
+  display: block;
+`);
```

Comment:
Better use the `sx` prop here
```suggestion
```

Reply:
Done. Corrections are made to both files (.js and .tsx)